.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
```

Comment:
[note]

I want to make sure this is okay.
Because both tab width 2 and 4 are used currently.
The first time I looked at the code, the tab width seemed to be 4.
But I noticed both 2 and 4 tab widths are used.
I followed the prettier default setting, tab width 2, here.

Reply:
yes 2 is good :+1: